tmt.spec

```diff
@@ -145,6 +145,7 @@ install -pm 644 bin/complete %{buildroot}/etc/bash_completion.d/tmt
 %files -n python%{python3_pkgversion}-%{name}
 %{python3_sitelib}/%{name}/
 %{python3_sitelib}/%{name}-*.egg-info/
+%{python3_sitelib}/schemas/
```

Comment:
@psss @lukaszachy @happz needs some guidance here, how to add the schema files to the rpm?

Reply:
Hm, I am not sure about RPM, but: `schemas` is an asset directory, not Python code, and I think we're missing a few pieces here after all:

* we're not letting Python packaging know these files should be included. We need to ask `setup.py` to include this directory in whatever packaging is made from tmt sources. With Poetry, we use `include`, https://gitlab.com/testing-farm/artemis/-/blob/main/server/pyproject.toml#L18, `setup.py` is using what, `package_data`? https://docs.python.org/3/distutils/setupscript.html#installing-package-data
* this would make sure Python is aware these files need to be bundled with tmt code. And maybe a specfile is able to work with this info, and include them in RPM? I don't know how exactly Python packaging with helpers works here :/
* the magic you do in the test to access the files - I believe this needs to be replaced with `pkg_resources`, see https://gitlab.com/testing-farm/artemis/-/blob/main/server/src/tft/artemis/__init__.py#L1183 - that should be native for Python, and it should work with all packaging flavors Python knows.
* and if there's no help from RPM macros, the files should be put manually into tmt's package directory, i.e. `lib64/pythonX.Y/site-packages/tmt/schemas` or similar. That's where `pkg_resources` would look for them, IIUIC.

Sorry for not realizing this sooner, but Python packaging should be aware of these files, otherwise TMT installations from pyPI would lack them. And I hope there are helpful macros for specfiles automagically including these files without bothering humans.

Reply:
> * `setup.py` is using what, `package_data`?

Yes, we were using `package_data` for including the `run.sh` script for the `detach` executor and it worked fine. It was removed in d6e260379b3d6d56f34b20399f42225db1bf1835. No spec file adjustment was needed.